packages/fxa-settings/src/components/Settings/ConnectedServices/index.stories.tsx

```diff
@@ -3,22 +3,45 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { Meta } from '@storybook/react';
+import { withLocalization } from '../../../../.storybook/decorators';
 import { LocationProvider } from '@reach/router';
 import { ConnectedServices } from '.';
 
-import { MOCK_SERVICES } from './mocks';
-import { AppContext } from 'fxa-settings/src/models';
+import { MOCK_SERVICES, MOCK_SERVICES_WITHOUT_MOBILE } from './mocks';
+import { Account, AppContext } from 'fxa-settings/src/models';
 import { mockAppContext } from 'fxa-settings/src/models/mocks';
```

Comment:
Only because I could see the previous file while 👀 this file, but I noticed above we're using:
```js
// relative `../../../`
import { AppContext } from '../../../models';
import { mockAppContext } from '../../../models/mocks';
```
Whereas here we're using:
```js
// absolute `fxa-settings/src/models/`
import { mockAppContext } from 'fxa-settings/src/models/mocks';
```

I can't imagine it matters since we're in the fxa-settings package already, but wasn't sure if one was preferable (since sometimes ../../.. can be tricky to easily parse.

Reply:
Good question, we seem to be using a mix of both throughout `fxa-settings`. My hunch is that VS Code has been auto-filling with relative paths and that's what we've mostly been using for convenience. If this does matter, we might want to file a follow-up ticket to update the paths more globally throughout the package.